### PR TITLE
JavaScript classes lesson: Remove Ruby mentions, update knowledge check link

### LINKED ISSUE #28015: Classes: First two links under 'Knowledge Check' refer to the same article.
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

update the second link to the correct article

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-classes

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by sukairaida**
## Because

The lesson talks about how JavaScript classes differ from other languages like Java or Ruby, but there is only a comparison with Java. The knowledge check link for the question about this also doesn't send the learners directly to where the differences between Java and JavaScript are explained.

## This PR

- Removes all mentions of Ruby
- Updates the knowledge check link to redirect directly to the image with the relevant information.

## Issue

Closes #28015

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/organizing_your_javascript_code/classes.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-JavaScript does *not* have classes in the same sense as other Object Oriented languages like Java or Ruby. ES6, however, *did* introduce a syntax for object creation that uses the `class` keyword. It is basically a new syntax that does the *exact* same thing as the object constructors and prototypes we learned about in the constructor lesson.
+JavaScript does *not* have classes in the same sense as other Object Oriented languages like Java. ES6, however, *did* introduce a syntax for object creation that uses the `class` keyword. It is basically a new syntax that does the *exact* same thing as the object constructors and prototypes we learned about in the constructor lesson.
```
Comment (MaoShizhong) [inline javascript/organizing_your_javascript_code/classes.md:3]: I actually think this particular mention is fine, especially accounting for the next comments

```suggestion
JavaScript does *not* have classes in the same sense as other object-oriented languages like Java or Ruby. ES6, however, *did* introduce a syntax for object creation that uses the `class` keyword. It is basically a new syntax that does the *exact* same thing as the object constructors and prototypes we learned about in the constructor lesson.
```

File: javascript/organizing_your_javascript_code/classes.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Since we've already gone fairly in-depth with Constructors, you don't have too m
 This section contains a general overview of topics that you will learn in this lesson.
 
 - Describe the pros and cons of using classes in JavaScript.
-- Briefly discuss how JavaScript's object creation differs from a language like Java or Ruby.
+- Briefly discuss how JavaScript's object creation differs from a language like Java.
```
Comment (MaoShizhong) [inline javascript/organizing_your_javascript_code/classes.md:14]: What do you think about rewording to the following or something similar? I think the LO and KC items would be better in a mire general frame. The current wording makes it feel like the point is to compare to Java, whereas the article in the KC part only uses Java as an example and focuses more on "JS is different to other things altogether"
```suggestion
- Briefly discuss how JavaScript's object creation differs from other object-oriented programming languages.
```

File: javascript/organizing_your_javascript_code/classes.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Go back to your [Library project](https://www.theodinproject.com/lessons/node-pa
 The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
 - [Describe the pros and cons of using classes in JavaScript.](https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65)
-- [How does JavaScript's object creation differ from a language like Java or Ruby?](https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65)
+- [How does JavaScript's object creation differ from a language like Java?](https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65#e6b3)
```
Comment (MaoShizhong) [inline javascript/organizing_your_javascript_code/classes.md:48]:
```suggestion
- [How does JavaScript's object creation differ from other object-oriented programming languages?](https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65#e6b3)
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

**Comment by sukairaida** [comment]
Sorry for the multiple commits with bad names, I just clicked "commit changes" here on the github page. Should I have done the changes locally into a single commit and then pushed? or was this fine?

**Review by MaoShizhong — APPROVED** [review]
It's all fine :) that commit suggestion feature is nice. If there are more.changes involved, especially a mix of those and ones I actually have to make manually as well, then I might prefer to do everything manually just so I don't need to sync my local branch with origin.
But it's fine either way.
Thanks for contributing!

**Pull request merged**